Assets/Scripts/Singletons/ClockManager.cs

```diff
@@ -26,9 +23,8 @@ public class ClockManager : MonoBehaviour
 
     private void Start()
     {
-		// Temporary
         UIManager.onCanvasActivated += StopClock;
-        UIManager.onCanvasDeactivated += StartClock; 
```

Comment:
Ignore this file. I originally fired event in UIManager passing a UICanvasType in, so changed the way the clockstopping worked, But ditched it now.